BraveShared/Analytics/DAU.swift

```diff
@@ -15,7 +15,7 @@ public class DAU {
     private static let baseUrl = "https://laptop-updates.brave.com/\(apiVersion)/usage/ios?platform=ios"
     
     /// Number of seconds that determins when a user is "active"
-    private let activeUserDuration = 10.0
+    private let pingRefreshDuration = 20.minutes
```

Comment:
We should change this to `5.minutes` now.
```suggestion
    private let pingRefreshDuration = 5.minutes
```